src/components/Numbers/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import dotsImage from './img/dots.png';
+import circleImage from './img/circle.png';
+import { charityAPI } from '../../clients';
+import './style.css';
+
+const Number = ({ number, title }) => {
```

Comment:
Good Job 👍 💯 
It is good practice to split your component into small component when it possible